### ISSUE TITLE
Do not affect manual actions with PAUSE

### ISSUE THREAD

**Issue by filimonov** [azat/chdig#51]
It seems like a valid scenario - pause updates, then dig deeper in one of the queries. But commands like explain etc don't work when the updates are paused.

**Comment by azat** [comment]
Yes indeed, I also hit this couple of times, but didn't bother to fix it, but since you found it, I guess I should :)
